packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_databases_general_information_add_node": "Ajouter un nœud",
+  "pci_databases_general_information_add_node_confirm": "Valider",
+  "pci_databases_general_information_add_node_cancel": "Annuler",
+  "pci_databases_general_information_add_node_description": "Vous disposez actuellement de {{nodesCount}} nœuds. Vous pouvez ajouter un nœud pour",
```

Comment:
Looks like the sentence is not finished here

Reply:
The price actually follows this sentence